openapi/tests/openapi_integration/test_snapshot.py

```diff
@@ -100,3 +105,101 @@ def test_snapshot_operations():
     assert len(response.json()['result']) == 0
 
 
+def test_test_snapshot_operations_non_wait():
+    # there no snapshot on collection
+    response = request_with_validation(
+        api='/collections/{collection_name}/snapshots',
+        method="GET",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
+    assert len(response.json()['result']) == 0
+
+    # create snapshot on collection
+    response = request_with_validation(
+        api='/collections/{collection_name}/snapshots',
+        method="POST",
+        path_params={'collection_name': collection_name},
+        query_params={'wait': 'false'},
+    )
+    assert response.status_code == 202
+
+    # wait for snapshot to be created
+    sleep(1)
+
+    # validate it exists
+    response = request_with_validation(
+        api='/collections/{collection_name}/snapshots',
+        method="GET",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
+    assert len(response.json()['result']) == 1
+    snapshot_name = response.json()['result'][0]['name']
+
+    # delete it
+    response = request_with_validation(
+        api='/collections/{collection_name}/snapshots/{snapshot_name}',
+        method="DELETE",
+        path_params={'collection_name': collection_name,
+                     'snapshot_name': snapshot_name},
+        query_params={'wait': 'false'},
+    )
+    assert response.ok
+    # wait for snapshot to be deleted
+    sleep(5)
```

Comment:
5 seconds seems a bit excessive IMHO

Was it not working with lower values?

Reply:
I must set the values to a minimum of 2 in order to pass the tests. If the values are any lower than 2, I will not meet the requirements for the tests.